webapp/components/Form.tsx

```diff
@@ -50,17 +50,25 @@ const today = new Date().toISOString().split('T')[0].toString();
 
 const localStorage =
   typeof window !== 'undefined' ? window.localStorage : undefined;
-const getFromLocalStorage = (key: keyof FormValues) =>
-  JSON.parse(localStorage?.getItem('formValues.' + key) ?? '""');
+const getFromLocalStorage = (key: keyof FormValues) => {
+  try {
+    return JSON.parse(localStorage?.getItem('formValues.' + key) ?? '""');
+  } catch (e) {}
+  return '';
+};
 const sessionStorage =
   typeof window !== 'undefined' ? window.sessionStorage : undefined;
 const getFromSessionStorage = (
   key: keyof FormValues,
   fallbackJsonString = '""'
-) =>
-  JSON.parse(
-    sessionStorage?.getItem('formValues.' + key) ?? fallbackJsonString
-  );
+) => {
+  try {
+    return JSON.parse(
+      sessionStorage?.getItem('formValues.' + key) ?? fallbackJsonString
+    );
+  } catch (e) {}
+  return '';
```

Comment:
I guess since the error is not used
```suggestion
  try {
    return JSON.parse(
      sessionStorage?.getItem('formValues.' + key) ?? fallbackJsonString
    );
  } catch () {
    return '';
  }
```

Reply:
`catch ()` is not allowed in TS. I could do `catch (_)`, but to me that doesn't really matter too much and e feels clearer anyhow.

Reply:
ah lol okay